src/discussions/posts/post/PostLink.jsx

```diff
@@ -84,7 +84,9 @@ function PostLink({
             </div>
           </div>
           <div className="text-truncate text-primary-500 font-weight-normal font-size-14 font-style-normal font-family-inter" style={{ 'max-height': '1.6em' }}>
-            <HTMLLoader htmlNode={post.renderedBody || post.rawBody} />
+            {isPostPreviewAvailable(post.previewBody)
+              ? <HTMLLoader htmlNode={post.previewBody} />
+              : <HTMLLoader htmlNode={intl.formatMessage(messages.postWithoutPreview)} />}
```

Comment:
preview body and postWithoutPreview both are plain text so we do not need to use the HTMLLoader component to display the string. Please use div or span.